app/controllers/application_controller/timelines.rb

```diff
@@ -298,9 +298,9 @@ def tl_build_init_options(refresh = nil)
 
     if @tl_options[:tl_show] == "policy_timeline"
       @tl_options[:all_results] = {}
-      @tl_options[:all_results]["Both"] = "both"
-      @tl_options[:all_results]["True"] = "success"
-      @tl_options[:all_results]["False"] = "failure"
+      @tl_options[:all_results][_("Both")] = "both"
+      @tl_options[:all_results][_("True")] = "success"
+      @tl_options[:all_results][_("False")] = "failure"
```

Comment:
`@tl_options[:all_results]` is a hash, which then becomes argument of `options_for_select`,
which interprets the hash so that the keys is what will be rendered in the ui, while the values
is what will be submitted during form post.